builtin/logical/pki/cert_util.go

```diff
@@ -1537,24 +1607,34 @@ func handleOtherSANs(in *x509.Certificate, sans map[string][]string) error {
 	return nil
 }
 
+const (
+	nameTypeEmail = 1
```

Comment:
Could you add a comment explaining where these values are coming from, with a link?

Reply:
Fixed.